synthetix-mintr/ru/translation.json

```diff
@@ -0,0 +1,416 @@
+      "burned": "Сожжено",
+      "claimedFees": "Полученные Вознаграждения",
+      "exchanged": "Куплено",
+      "traded": "Торговый оборот",
+      "sent": "Отправлено",
+      "deposited": "На депозите",
+      "withdrawn": "Снято",
+      "sold": "Продано"
+    }
+  },
+  "transactionSettings": {
+    "priceIndicator": "Комиссионные сети Ethereum:",
+    "title": "Установите цену газа",
+    "subtitle": "Установите цену газа (GWEI), чтобы задать скорость транзакции.",
+    "minutes": "мин.",
+    "button": {
+      "submit": "Подтвердить"
+    }
+  },
+  "transactionProcessing": {
+    "confirmation": {
+      "title": "Пожалуйста, подтвердите транзакцию",
+      "subtitle": "Чтобы продолжить, следуйте инструкциям в вашем <1>walletType</1> Кошельке.",
+      "loading": "Ожидание действий пользователя..."
+    },
+    "complete": {
+      "subtitle": "Ваша транзакция отправлена в сеть Ethereum. Вы можете закрыть это окно, транзакция завершится в фоновом режиме."
+    },
+    "error": {
+      "title": "Что-то пошло не так...",
+      "subtitle": "Код:",
+      "type": {
+        "userDenied": "Пользователь не подписал транзакцию",
+        "generic": "Ошибка"
+      }
+    }
+  },
+  "notification": {
+    "status": {
+      "pending": "Транзакция в процессе",
+      "success": "Транзакция завершена!",
+      "error": "Произошла ошибка..."
+    }
+  },
+  "button": {
+    "max": "Макс.",
+    "burnMax": "Сжечь максимум",
+    "fixCRatio": "Исправьте ваш Залоговый Коэффициент",
+    "edit": "Редактировать",
+    "havingTrouble": "Возникли проблемы?",
+    "whatIsSynthetix": "Что такое Synthetix?",
+    "previous": "Назад",
+    "next": "Дальше",
+    "startOver": "В начало",
+    "navigation": {
+      "cancel": "Отмена",
+      "back": "Назад",
+      "close": "Закрыть",
+      "etherscan": "Посмотреть на Etherscan",
+      "view": "Посмотреть",
+      "finish": "Завершить и вернуться на главную",
+      "ok": "ОК"
+    }
+  },
+  "input": {
+    "list": {
+      "placeholder": "Поиск синта..."
+    },
+    "error": {
+      "notEnoughToMint": "В данный момент вы не можете выпустить столько sUSD",
+      "gasEstimate": "Ошибка при оценке газа",
+      "invalidAmount": "Недействительная сумма",
+      "notEnoughToBurn": "У вас нет такого количества sUSD для сжигания. Вы можете купить больше sUSD на dex.ag по лучшей цене.",
+      "balanceTooLow": "Ваш баланс слишком низкий",
+      "lowerThanMinDeposit": "Сумма недостаточна для минимального депозита"
+    }
+  },
+  "general": {
+    "noData": "Нет данных"
+  },
+  "network": {
+    "gas": "Комиссионные сети Ethereum (Gas):",
+    "tradingFee": "Комиссионные обмена Синтов:",
+    "speed": "Ориентировочная скорость транзакции:"
+  },
+  "maintenance": {
+    "title": "Mintr обновляется и недоступен в данный момент.",
+    "subtitle": "Приносим извинения за неудобства, скоро все наладится."
+  },
+  "mobileLanding": {
+    "title": "Mintr is currently unavailable for mobile and tablet.",
```

Comment:
@ivanaka you forgot some translations here ;)

Reply:
Fixed missed entries